### ISSUE TITLE
create issue template for I want to become a reviewer

### ISSUE THREAD

**Issue by katjaq** [brainhack-proceedings/brainhack-proceedings.github.io#7]
including 
* field of research
* coding language
very much like in joss

**Comment by lunebellec** [comment]
at the moment we have a google form, which I think was set up by @agahkarakuzu 

I agree with @katjaq that using an issue template would be great, because it avoids expanding our tech stack, and everything lives neatly under a single github repo. Let me know what you think @agahkarakuzu and others.

**Comment by agahkarakuzu** [comment]
@pbellec I agree that keeping 3rd part integrations at a minimum is better in the long run. The whole process is boiled down to a single link anyway :)

**Comment by complexbrains** [comment]
I will create one, will assign it to myself it everyone agrees?

**Comment by lunebellec** [comment]
the only downside of using an issue template I can think of is that it makes the application public. But the whole review process is transparent anyway, and I don't really expect we will turn down individuals. It as always possible to contact them through private channels if we think it's preferrable, so I don't think it will be a problem.

**Comment by complexbrains** [comment]
Given the scope is HOPE we might set up a couple of soft criteria like affiliation, 1-2 paper in the field/tutorial they published&run/contributions in the event organization, educational activities and time allocation they can promise us to help with to avoid having too many random applications maybe and can indicate there might be an evaluation at some point by the editorial team regarding the need of reviewer pool at that period of time. If that makes sense?

**Comment by katjaq** [comment]
Maybe we can have an easy approach and keep it very welcoming like we love it until we should really face a problem?
Like Hacktoberfest --> for so many years they successfully super openly ran 1 month of world wide collaborations – this year they got spammed and upon that reacted to be more strict (i.e. inclusive upon check).

If we ask people for the neuroscientific topic they are into and feel comfortable reviewing, (and maybe for code languages they feel most comfortable with ((maybe we want to not ask for that in case it may make people think oooo these brainhackers all about hard core coding and hacking?)) ) we could have a nice starting point of motivated people who'd like to invest time for the community <3 

This way, reviewer adding can be without human intervention also :) keeping our workload low (which we may appreciate once Brainhack Proceedings takes off :rocket: )
